OwinoLucas/I-Reporter
# installed DRF-jwt and used it to create a user auth endpoint returns a token with the payload

## What a the PR does
* The pull request verifies that the login endpoint works
## Description
* The pull request shows the serialization of login endpoints where a user enters his/her credentials and the server's response is the users full name along with a token.
## How to test Pull request
* Follow instructions in the README.md to acquire the project in your computer.
* Ensure local server is running, open postman and feed the link below to make the POST request.
```bash
http://127.0.0.1:8000/login/
```
## Screenshots
![Screenshot from 2020-06-18 17-25-39](https://user-images.githubusercontent.com/60548928/85032729-d5de3680-b188-11ea-9775-7f2cb9560663.png)

## Trello task link
[https://trello.com/c/FUthAFiy](link)
